topics/statistics/tutorials/RNN/tutorial.md

```diff
@@ -0,0 +1,368 @@
+  feed to the network, the training will be very slow (as we have 25000 training examples in our dataset). To speed up the training, we present 
+  only a subset of the training examples to the network, after which we update the weights/biases. *batch_size* decides the size of this subset. 
+  The model builder can be downloaded as a zip file.
+{: .comment}
+
+### **Deep learning training and evaluation**
+
+> ### {% icon hands_on %} Hands-on: Training the model
+>
+> 1. **Deep learning training and evaluation** conduct deep training and evaluation either implicitly or explicitly {% icon tool %} with the following parameters:
+>    - *"Select a scheme"*: `Train and Validate`
+>    - *"Choose the dataset containing pipeline/estimator object"*: Select the *Keras Model Builder* from the previous step. 
+>    - *"Select input type:"*: `tabular data`
+>        - *"Training samples dataset"*: Select `X_train` dataset
+>        - *"Choose how to select data by column:"*: `All columns`
+>        - *"Dataset containing class labels or target values"*: Select `y_train` dataset
+>        - *"Choose how to select data by column:"*: `All columns`
+>    - Click *"Execute"*
+>
+>
+{: .hands_on}
+
+> ### {% icon comment %} Comment
+> The training step generates 3 datasets. 1) accuracy of the trained model, 2) the trained model, downloadable as a zip file, and 3) the trained 
+  model weights, downloadable as an hdf5 file. These files are needed for prediction in the next step. 
+{: .comment}
+
+### **Model Prediction**
+
+> ### {% icon hands_on %} Hands-on: Testing the model
+>
+> 1. **Model Prediction** predicts on new data using a prefitted model {% icon tool %} with the following parameters:
+>    - *"Choose the dataset containing pipeline/estimator object"* : Select the trained model from the previous step. 
+>    - *"Choose the dataset containing weights for the estimator above"* : Select the trained model weights from the previous step. 
+>    - *"Select invocation method"*: `predict`
+>    - *"Select input data type for prediction"*: `tabular data`
+>    - *"Training samples dataset"*: Select `X_test` dataset 
+>    - *"Choose how to select data by column:"*: `All columns`
+>    - Click *"Execute"*
+>
+{: .hands_on}
+
+> ### {% icon comment %} Comment
+> The prediction step generates 1 dataset. It's a file that has predictions (1 or 0 for positive or negative movie reviews) for every review in 
+  the test dataset.
+{: .comment}
+
+### **Machine Learning Visualization Extension**
+
+> ### {% icon hands_on %} Hands-on: Creating the confusion matrix
+>
+> 1. **Machine Learning Visualization Extension** includes several types of plotting for machine learning {% icon tool %} with the following parameters:
+>    - *"Select a plotting type"*: `Confusion matrix for classes`
+>    - *"Select dataset containing the true labels"*": `y_test` 
+>    - *"Choose how to select data by column:"*: `All columns`
+>    - *"Does the dataset contain header:"*: `Yes`
```

Comment:
I think a step to include a dataset of `predicted labels` is missing. Plotting a confusion matrix would require true and predicted both datasets.

Reply:
Added the step.